recipes-core/packagegroups/packagegroup-ni-base.bb

```diff
@@ -15,94 +15,90 @@ inherit packagegroup
 MACHINE_ESSENTIAL_EXTRA_RDEPENDS ?= ""
 MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS ?= ""
 
-NILRT_x64_PACKAGES = "\
-	dmidecode \
-	e2fsprogs \
-	e2fsprogs-mke2fs \
-	efivar \
-	fw-printenv \
-	linux-firmware-i915 \
-	nilrtdiskcrypt \
-	phc2sys \
-	pstore-save \
-"
 
-NILRT_PACKAGES = "\
-	busybox-ifplugd \
-	busybox-udhcpd \
-	busybox-zcip \
-	glibc-gconv-utf-16 \
-	init-ifupdown \
-	libstdc++ \
-	logrotate \
-	niwatchdogpet \
-	openvpn \
-	pigz \
-	usbutils \
-	${@bb.utils.contains('COMBINED_FEATURES', 'pci', 'pciutils-ids', '',d)} \
-	${NILRT_x64_PACKAGES} \
+RDEPENDS_${PN} = "\
+	${MACHINE_ESSENTIAL_EXTRA_RDEPENDS} \
+	${VIRTUAL-RUNTIME_mountpoint} \
 "
 
-RDEPENDS_${PN} = "\
+RDEPENDS_${PN} += "\
+	${@bb.utils.contains('COMBINED_FEATURES', 'pci', 'pciutils-ids', '',d)} \
+	${@bb.utils.contains('MACHINE_FEATURES', 'acpi', 'busybox-acpid', '', d)} \
+	${@bb.utils.contains('MACHINE_FEATURES', 'keyboard', 'keymaps', '', d)} \
 	avahi-daemon \
 	base-files \
 	base-files-nilrt \
 	base-passwd \
-	${@bb.utils.contains('MACHINE_FEATURES', 'keyboard', 'keymaps', '', d)} \
 	busybox \
-	${@bb.utils.contains('MACHINE_FEATURES', 'acpi', 'busybox-acpid', '', d)} \
+	busybox-ifplugd \
+	busybox-udhcpd \
+	busybox-zcip \
 	coreutils-hostname \
 	crio-support-scripts \
 	cronie \
 	curl \
 	daemonize \
 	dkms \
+	dmidecode \
 	dpkg-start-stop \
+	e2fsprogs \
+	e2fsprogs-mke2fs \
+	efivar \
 	ethtool \
+	eudev \
+	fw-printenv \
+	glibc-gconv-utf-16 \
 	gptfdisk-sgdisk \
+	init-ifupdown \
 	initscripts \
 	initscripts-nilrt \
 	iproute2 \
 	iptables \
-	kmod \
 	kernel-modules \
+	kmod \
 	libavahi-client \
 	libavahi-common \
 	libavahi-core \
 	libcap-bin\
 	libnss-mdns \
 	libpam \
 	librtpi \
+	libstdc++ \
+	linux-firmware-i915 \
```

Comment:
Did we accidentally drop the radeon firmware in Hardknott? It was here in Sumo: https://github.com/ni/meta-nilrt/blob/e7eb865054449d1b104f19fc4a376da84217cd03/recipes-core/packagegroups/packagegroup-ni-base.bb#L25

Reply:
We intentionally moved it to the runmode packagegroup [here](https://github.com/ni/meta-nilrt/commit/dbc1d0ce84e33a22fbf67a2e9c0d164d1ad46435).

Reply:
Ah, yes. Sorry, I forgot about that.

Reply:
Having only the i915 firmware in there threw me off. Now I seem to recall commenting on the other commit you've linked that we should look into if we can make the i915 firmware runmode only too. I think the answer is yes but we need to do some testing to make sure.

Reply:
A follow-on work item for moving the i915 package was created [here](https://ni.visualstudio.com/DevCentral/_workitems/edit/1745937).